pkgs/development/guile-modules/guile-reader/default.nix

```diff
@@ -13,7 +13,15 @@ stdenv.mkDerivation rec {
     sha256 = "0592s2s8ampqmqwilc4fvcild6rb9gy79di6vxv5kcdmv23abkgx";
   };
 
-  patches = [ ./0001-fix-prototypes.diff ];
+
+
+
+
+  patches = [
+     fetchurl {
```

Comment:
You will need to wrap it in parentheses: see https://github.com/NixOS/nixpkgs/blob/56904d7c423f2b13b37fbd29f39bbb4b52bc7824/pkgs/games/holdingnuts/default.nix#L19-L22

Reply:
ok!